lib/recurly/push_notification.php

```diff
@@ -21,27 +21,29 @@ class Recurly_PushNotification
    *    failed_payment_notification
    *    successful_refund_notification
    *    void_payment_notification
+   *    *_invoice_notification
```

Comment:
Do you mind spelling out the notification types? We do that with the others and I think it'd be good to follow the pattern:
```
new_invoice_notification
closed_invoice_notification
past_due_invoice_notification
```